sdk/playwrighttesting/microsoft-playwright-testing/test/core/playwrightService.spec.ts

```diff
@@ -28,6 +29,7 @@ describe("getServiceConfig", () => {
     sandbox.stub(console, "log");
     process.env[ServiceEnvironmentVariable.PLAYWRIGHT_SERVICE_URL] =
       "wss://eastus.playwright.microsoft.com/accounts/1234/browsers";
+    process.env[InternalEnvironmentVariables.MPT_PLAYWRIGHT_VERSION] = "1.47.0";
```

Comment:
NIT: can we avoid multiple entry for this 1.47 string and instead use some common constant